examples/nestjs/consumer/test/pact/pact-consumer-config-options.service.ts

```diff
@@ -0,0 +1,35 @@
+import * as path from 'path';
+import { Injectable } from '@nestjs/common';
+import {
+  PactConsumerOptionsFactory,
+  PactConsumerOverallOptions,
+} from 'nestjs-pact';
+
+@Injectable()
+export class PactConsumerConfigOptionsService
+  implements PactConsumerOptionsFactory {
+  createPactConsumerOptions(): PactConsumerOverallOptions {
```

Comment:
As discussed, I think pact's config generally can be improved. 

However, this file includes consumer test config and broker config, which are concepts that aren't needed at the same time, so I think it would be clearer to separate them into their own config files. I think they could be plain objects.